tests/playwright-test/types.spec.ts

```diff
@@ -71,12 +71,6 @@ test('should check types of fixtures', async ({ runTSC }) => {
         // @ts-expect-error
         baz: true,
       });
-      const fail9 = test.extend<{ foo: string }>({
```

Comment:
This doesn't produce an error anymore - @dgozman do you know why we added this test case in the first place? Since `auto` fixtures for the `test` scope seem to work as well:

```ts
import { test as base } from '@playwright/test';

const test = base.extend<{ foo: string }>({
  foo: [async ({ }, use) => {
    console.log('hello from the fixture')
    await use('foo');
  }, { scope: 'test', auto: true }],
})

test('my test', async ({ }) => {
  console.log('hello from the test')
});
```

Reply:
This is because we do not allow overriding a non-auto fixture with an auto-fixture. I wonder why does it work now?

Reply:
Proposed change [in the types](https://github.com/microsoft/playwright/blob/main/utils/generate_types/overrides-test.d.ts#L147-L149) to fix the problem:

```ts
  [K in Exclude<keyof W, keyof PW>]?: [WorkerFixtureValue<W[K], W & PW>, { scope: 'worker', auto?: boolean, option?: boolean, timeout?: number | undefined, title?: string, box?: boolean }];
} & {
  [K in Exclude<keyof T, keyof PT>]?: TestFixtureValue<T[K], T & W & PT & PW> | [TestFixtureValue<T[K], T & W & PT & PW>, { scope?: 'test', auto?: boolean, option?: boolean, timeout?: number | undefined, title?: string, box?: boolean }];
```

Reply:
Awesome!